addons/mass_mailing/models/mailing.py

```diff
@@ -132,6 +138,9 @@ def _get_default_mail_server_id(self):
     mailing_domain = fields.Char(
         string='Domain', compute='_compute_mailing_domain',
         readonly=False, store=True)
+    is_outgoing_server_enabled = fields.Boolean(
```

Comment:
Maybe rename to mail_server_available ? So that it uses the same namespacing as mail_server_id.

Reply:
Renamed to mail_server_available.